Helios/HeliosConfig.h

```diff
@@ -1,6 +1,23 @@
 #ifndef HELIOS_CONFIG_H
 #define HELIOS_CONFIG_H
 
+// Helper macros to expand out the version into a string
+#define _STR(x) #x
+#define STR(x) _STR(x)
+
+// Helios Version Number
+//
+// It is expected that HELIOS_VERSION will be provided on the command line to the
+// compiler as a -DHELIOS_VERSION=a.b.c but if it's not it will default here
+#ifndef HELIOS_VERSION
+#define HELIOS_VERSION 0.0.1

```

Comment:
you define -DHELIOS_VERSION_NUMBER but this is HELIOS_VERSION those need to match.